### DISCUSSION TITLE
Frigate is fantastic

### DISCUSSION THREAD

**Discussion by Dan1001** [blakeblackshear/frigate#6401] [General]
I just wanted to thank the devs. I've tried everything: zoneminder, shinobi, xeoma, ispy, AgentDVR. Frigate has the perfect balance of performance, reliability and simplicity. I spent so long fiddling with the others to make them work, and it always felt like they were on the edge of falling over. Frigate just does its job.

Thank you!

**Comment by lienrag35** [comment]
Fully agree with you  :)
